dvc/repo/run.py

```diff
@@ -74,10 +78,12 @@ def run(self, fname=None, no_exec=False, single_stage=False, **kwargs):
 
     dvcfile = Dvcfile(self, stage.path)
     if dvcfile.exists():
-        if stage_name and stage_name in dvcfile.stages:
+        if kwargs.get("overwrite", True):
+            dvcfile.remove_stage(stage)
```

Comment:
There's no need for prompt here imo. Users can simply add `--overwrite` and re-run, as to this point, we are not doing anything substantial.

Reply:
@skshetry Agreed! This is in-line with `no-prompt` policy that is reflected in many tickets.